### ISSUE TITLE
Multiple defaultValues for a matrix

### ISSUE THREAD

**Issue by zhnedyalkow** [surveyjs/survey-library#2550]
### Are you requesting a feature, reporting a bug or asking a question?

Asking a question

### What is the current behavior?

I would like to know if it is possible to apply **multiple defaultValues** for **matrix** question? For example instead specify question value and answer value, to have multiple answer values in list (['6', '7']). How such behaviour can be achieved in SurveyJs library?

### What is the expected behavior?

To have ability to apply multiple values for a matrix question

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```

const survey = {
               ....,   
               defaultValue: {'1836': ['6', '7'], '1832': '7'}, 
               rows: [
                {
                  questionId: 994,
                  questionAnswerStackId: 1836,
                  text: 'Example text',
                  value: '1836',
                  position: 1,
                }, 
                {
                  questionId: 990,
                  text: 'Example text2',
                  value: '1832',
                  position: 1,
              }],
              columns: [  
                { answerId: 10, text: 'Disagree', value: '10', order: 1, isCorrect: false},
                { answerId: 9, text: 'Somewhat Disagree', value: '9', order: 2, isCorrect: false},
                { answerId: 7, text: 'Somewhat Agree', value: '7', order: 3, isCorrect: true},
                { answerId: 6, text: 'Agree', value: '6', order: 4, isCorrect: true}
              ]

```

### Specify your

* browser:
* browser version:
* surveyjs platform (angular or react or jquery or knockout or vue): react
* surveyjs version: 1.7.18

**Comment by andrewtelnov** [comment]
@zhnedyalkow I am not sure I understnad you correctly. There is "defaultRowValue" property in matrix dynamic question. We do not have it in matrix dropdown. Are you looking for this functionality?

Thank you,
Andrew

**Comment by zhnedyalkow** [comment]
Hi @andrewtelnov,

I have a single choice matrix  ("type": "matrix" ) and would like to show multiple positive scores (answers). It appears that 'defaultValue' accept only a single value (answer) for a specific question:

(defaultValue: {'1832': '7'},  but cannot pass multiple answers -> defaultValue: {'1832': ['6', '7']}). 

1) Is it possible to have multiple positive answers for a single question? 
2) If not what is the proper way to handle it in order to have such functionality?
3) Could you please provide a sandbox/showcase if it is possible?

Than you in advance!

BR,
Zhitomir

**Comment by zhnedyalkow** [comment]
In addition to this, the business case is that the user needs to have ability to preview/see the positive/correct answers for each question in the single choice matrix.

**Comment by andrewtelnov** [comment]
@zhnedyalkow You can use multiple dropdown matrix with `showInMultipleColumns: true` property in checkbox column. Here is [the example](https://plnkr.co/edit/BVPjbAiAEy9O1Y9t).

Thank you,
Andrew

**Comment by zhnedyalkow** [comment]
Hi @andrewtelnov,

Thank you very much for your prompt response.

It seems that the proposed solution will solve my issue, but I am not able to disable clicking on the checkboxes. 

I reviewed the documentation and tried to use 'readOnly: true' prop, but it seems that it is not working for 'matrixdropdown'.

Am I going into right direction?

Best regards,
Zhitomir

**Comment by andrewtelnov** [comment]
Continued [here](https://github.com/surveyjs/survey-library/issues/2560).
There was a bug in react version with checkbox column when property `showInMultipleColumns: true` set to true and column (question or survey) is read-only. We fixed it in [v1.8.19](https://surveyjs.io/WhatsNew#v1.8.19)

Andrew

**Comment by zhnedyalkow** [comment]
@andrewtelnov thank you, you can close this ticket for now.
